src/main/webapp/app/quiz-statistics/quiz-statistics-footer/quiz-statistics-footer.component.ts

```diff
@@ -0,0 +1,205 @@
+import { Component, OnInit, OnDestroy, Input } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { QuizExercise, QuizExerciseService } from 'app/entities/quiz-exercise';
+import { QuizStatisticUtil } from 'app/components/util/quiz-statistic-util.service';
+import { QuizQuestion, QuizQuestionType } from 'app/entities/quiz-question';
+import { MultipleChoiceQuestionStatistic } from 'src/main/webapp/app/entities/multiple-choice-question-statistic';
+import { ShortAnswerQuestionUtil } from 'app/components/util/short-answer-question-util.service';
+import { QuizPointStatistic } from 'src/main/webapp/app/entities/quiz-point-statistic';
+import { ArtemisMarkdown } from 'app/components/util/markdown.service';
+import { AccountService, JhiWebsocketService } from 'app/core';
+import { TranslateService } from '@ngx-translate/core';
+import { Subscription } from 'rxjs/Subscription';
+import { HttpResponse } from '@angular/common/http';
+import * as moment from 'moment';
+
+@Component({
+    selector: 'jhi-quiz-statistics-footer',
+    templateUrl: './quiz-statistics-footer.component.html',
+    providers: [QuizStatisticUtil, ShortAnswerQuestionUtil, ArtemisMarkdown],
+})
+export class QuizStatisticsFooterComponent implements OnInit, OnDestroy {
+    @Input() isQuizPointStatistic: boolean;
+    @Input() isQuizStatistic: boolean;
+
+    readonly DRAG_AND_DROP = QuizQuestionType.DRAG_AND_DROP;
```

Comment:
Here is the same case as above of assigning enum members as constants.
I think it's better to just the enum.